_topics/en/psbt.md

```diff
@@ -132,6 +132,12 @@ optech_mentions:
   - title: "LND #4389 adds a new `psbt` wallet subcommand for creating & signing PSBTs"
     url: /en/newsletters/2020/10/07/#lnd-4389
 
+  - title: A new backwards incompatible version of PSBT is proposed
```

Comment:
backwards-incompatible version (needs the hyphen, otherwise backwards modifies version)

Reply:
(sorry for the late review, I only now realised that I didn't look at the topics changes)

Reply:
Fixed, thanks!